papermill/execute.py

```diff
@@ -56,7 +57,7 @@ def execute_notebook(
         Duration in seconds to wait for kernel start-up
     report_mode : bool, optional
         Flag for whether or not to hide input.
-    cwd : str, optional
+    cwd : str or Path, optional
```

Comment:
@willingc Are there other public APIs we might want to make this modification to?

Reply:
Let me take a closer look tomorrow. Thanks @drhagen.

Reply:
Maybe `inspect_notebook` in `papermill/inspection.py`

Reply:
Nevermind, @drhagen got that below.